src/pip/_internal/utils/misc.py

```diff
@@ -764,16 +765,42 @@ def unpack_file(
         )
 
 
+def make_command(*args):
+    # type: (Union[str, HiddenText]) -> CommandArgs
+    command_args = []  # type: CommandArgs
```

Comment:
Alternatively: `return cast('CommandArgs', args)`

Reply:
Let's see if `make_command()` changes first. I do like that using code rather than a cast protects us from mistakes in case anything about `HiddenText` or `CommandArgs` changes..